modules/tensor_mechanics/test/tests/domain_integral_thermal/interaction_integral_2d.i

```diff
@@ -69,71 +69,70 @@
 []
 
 [Modules/TensorMechanics/Master]
-  [./master]
+  [all]
     strain = FINITE
     add_variables = true
     incremental = true
```

Comment:
incremental = true is the default for finite

Reply:
I'll remove this